### PR TITLE
Changes to close issue #245 & #253

### PR DESCRIPTION
**Pull request by JamiePhonic**
This implements some minor changes which should close issues #245 & #253.

### CODE REVIEW

**Comment by zaza7** [comment]
The V5 rewrite for music already has the staying in voice after the playlist ends option, eslint corrections imo are minimal, for purge that's not the correct way to delete the command message as if you then attempt to prune 100 messages it will add 1 thereby requesting 101 messages from the endpoint and causing an error since only 100 messages can be gotten at any single time.

**Comment by JamiePhonic** [comment]
Making the bot stay in voice after the playlist ends was an easy fix, so i thought i'd just include it anyway
and i've modified my last commit to  fix the changes i made to purge.

**Comment by Nenkai** [comment]
I guess this is a good addition while we wait for V5, wasn't too hard to do anyway. Looks fine to me.

File: runtime/commands/default.js [hunk 1]
```diff
@@ -90,7 +90,14 @@ Commands.purge = {
               deleteMe.push(result.messages[x])
             }
           }
-          msg.channel.sendMessage(`${deleteMe.length} message(s) have been purged. ${cantDelete} were omitted due to them being over two weeks old.`)
+          msg.channel.sendMessage(`${deleteMe.length} message(s) have been purged. ${cantDelete} were omitted due to them being over two weeks old.`).then((m) => {
+            if (config.settings.autodeletemsg) {
+              setTimeout(() => {
+                m.delete().catch((e) => Logger.error(e))
+              }, config.settings.deleteTimeout)
+            } else {
```
Comment (zaza7) [inline runtime/commands/default.js]: This else block can be safely removed.

File: runtime/commands/default.js [hunk 2]
```diff
@@ -90,7 +90,14 @@ Commands.purge = {
               deleteMe.push(result.messages[x])
             }
           }
-          msg.channel.sendMessage(`${deleteMe.length} message(s) have been purged. ${cantDelete} were omitted due to them being over two weeks old.`)
+          msg.channel.sendMessage(`${deleteMe.length} message(s) have been purged. ${cantDelete} were omitted due to them being over two weeks old.`).then((m) => {
+            if (config.settings.autodeletemsg) {
+              setTimeout(() => {
+                m.delete().catch((e) => Logger.error(e))
+              }, config.settings.deleteTimeout)
```
Comment (zaza7) [inline runtime/commands/default.js]: Why not use deleteTimeoutLong instead of deleteTimeout

File: config.example.json [hunk 3]
```diff
@@ -16,6 +16,9 @@
     "deleteTimeoutLong": 6000,
     "maxvcslots": 10
   },
+  "voice":{
+    "leaveAfterEndOfPlaylist": false
```
Comment (zaza7) [inline config.example.json]: This can be added to the settings property instead of making a new prop.

File: runtime/internal/voice.js [hunk 4]
```diff
@@ -182,9 +182,20 @@ function next (msg, suffix, bot) {
       if (connection.voiceConnection.guild.id === msg.guild.id) {
         if (list[msg.guild.id].link.length === 0) {
           delete list[msg.guild.id]
```
Comment (zaza7) [inline runtime/internal/voice.js]: Not sure if you tested this but this delete should only run when the bot has left voice not if it's staying.

**Comment by JamiePhonic** [comment]
Thanks @zaza7 
I've implemented the changes you suggested. 
I wasn't aware that "no indentation on chained methods" was actually an IDE config thing. I'll remember that for next time!

**Pull request merged**